src/bootstrap/bootstrap.py

```diff
@@ -894,12 +894,18 @@ def bootstrap(help_triggered):
     build.clean = args.clean
 
     try:
-        toml_path = os.getenv('RUST_BOOTSTRAP_CONFIG') or args.config or 'config.toml'
-        if not os.path.exists(toml_path):
-            toml_path = os.path.join(build.rust_root, toml_path)
-
-        with open(toml_path) as config:
-            build.config_toml = config.read()
+        # Read from `RUST_BOOTSTRAP_CONFIG`, then `--config`, then fallback to `config.toml` (if it
+        # exists).
+        toml_path = os.getenv('RUST_BOOTSTRAP_CONFIG') or args.config
+        if not toml_path and os.path.exists('config.toml'):
+            toml_path = 'config.toml'
+
+        if toml_path:
+            if not os.path.exists(toml_path):
+                toml_path = os.path.join(build.rust_root, toml_path)
+
+            with open(toml_path) as config:
+                build.config_toml = config.read()
     except (OSError, IOError):
```

Comment:
Hm, this is pre-existing, I think, but this actually feels like maybe it's too eagerly ignoring errors here? Certainly in the case that `--config` or `RUST_BOOTSTRAP_CONFIG` is specified we shouldn't ignore failures to read the toml_path.

Reply:
Pushed a commit that removes this `try`/`except` block.